gensim/models/atmodel.py

```diff
@@ -107,104 +152,69 @@ def construct_author2doc(doc2author):
-        name.
-
-        Example:
-
+        API for Author-Topic model.
+
+        Parameters
+        ----------
+        corpus : iterable of iterable of (int, int)
+            Corpus of documents in appropriate format(BoW, UCI etc).
+        num_topics : int, optional
+            Number of topics to be extracted from the training corpus.
+        id2word : dict of {int: str}, optional
+            A mapping from word ids (integers) to words (strings).
+        author2doc : dict of {str: list of int}
+            A dictionary where keys are the names of authors and values are lists of
+            documents that the author contributes to.
+        doc2author : dict of {int: list of str}
+            A dictionary where the keys are document IDs and the values are lists of author names.
+        passes : int
+            Number of times the model makes a pass over the entire training data.
+        iterations : int
+            Maximum number of times the model loops over each document
+        chunksize : int
+            Controls the size of the mini-batches.
+        alpha : float
+            Hyperparameters for author-topic model.Supports special values of 'asymmetric'
+            and 'auto': the former uses a fixed normalized asymmetric 1.0/topicno prior,
+            the latter learns an asymmetric prior directly from your data.
+        eta : float
+            Hyperparameters for author-topic model.
+        update_every : int
+            Make updates in topic probaility for latest mini-batch.
+        eval_every : int
+            Calculate and estimate log perplexity for latest mini-batch.
+        gamma_threshold : float
+            Threshold value of gamma(topic difference between consecutive two topics)
+            until which the iterations continue.
+        decay : float
+            Controls how old documents are forgotten.
+        offset : float
+            Controls down-weighting of iterations.
+        minimum_probability : float
+            Controls filtering the topics returned for a document (bow).
+        random_state : int or a numpy.random.RandomState object.
+            Set the state of the random number generator inside the author-topic model.
+        serialized : bool
+            Indicates whether the input corpora to the model are simple lists
+            or saved to the hard-drive.
+        serialization_path : str
+            Must be set to a filepath, if `serialized = True` is used.
+
+        Example
+        -------
+        >>> import numpy as np
+        >>> from gensim.models import AuthorTopicModel
         >>> model = AuthorTopicModel(corpus, num_topics=100, author2doc=author2doc, id2word=id2word)  # train model
-        >>> model.update(corpus2)  # update the author-topic model with additional documents
+        >>> model.update(corpus, author2doc)  # update the author-topic model with additional documents
```

Comment:
Example **must** work, `corpus`, `author2doc`, `id2word` undefined here.

Reply:
Also, "update with additional", but you use exactly same `corpus` here, this confused